.changeset/olive-snakes-judge.md

```diff
@@ -0,0 +1,6 @@
+---
+"@clerk/clerk-expo": minor
+---
+
+**Breaking**: Export `useLocalCredentials` from a submodule.
```

Comment:
❓  Is it ok to introduce a breaking change in a `minor` release?

Reply:
@anagstef , the breaking change was already released in 3663 as a minor release. This was presumably by accident, due to unexpected expectations of how tree shaking would work with metro. So, technically no, its not ok but its already done; but it wasn't intended to be breaking either.
This fix is the "correction" for that. If this 'fixes' the break, then the feature itself is indeed nonbreaking, so this would be a patch.

Reply:
To be clear: this PR would be a patch and the *feature* would be (once corrected ) a minor since its not an intentional breaking change requiring a migration, and doesn't deprecate any  surface of the api.

Reply:
I'd have to agree that this is a breaking change, but it needs to be released as minor is order to fix a bug + introduces an API change.